luaui/Widgets/gui_buildmenu.lua

```diff
@@ -736,11 +736,6 @@ local function drawCell(cellRectID, usedZoom, cellColor, progress, highlightColo
 		font2:Print("\255\175\175\175" .. text, cellRects[cellRectID][1] + cellPadding + (cellInnerSize * 0.05), cellRects[cellRectID][4] - cellPadding - priceFontSize, priceFontSize * 0.82, "o")
 	end
 
-	-- draw build progress pie on top of texture
-	if progress and showBuildProgress then
```

Comment:
`progress` is always passed as nil

Reply:
Does the clock reveal for build progress still draw on top of the tile during construction?

Reply:
Yes, it's done outside the scope of this function